.babelrc

```diff
@@ -0,0 +1,20 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript",
+    "@babel/preset-react",
+    "@emotion/babel-preset-css-prop",
+    "minify"
```

Comment:
let's not minify, bundlers does it and it's convenient to read the code from node modules

Reply:
I have removed it in https://github.com/Open-Attestation/react-component-template/pull/25/commits/20e36968d046a0dbf7fa53cba9ae3197ea42308b.